src/lib/spinel/spinel_decoder.cpp

```diff
@@ -72,7 +72,7 @@ otError Decoder::ReadBool(bool &aBool)
     otError error = OT_ERROR_NONE;
     uint8_t byte;
 
-    SuccessOrExit(error = ReadUint8(byte));
+    LibUtils_SuccessOrExit(error = ReadUint8(byte));
```

Comment:
@Irving-cl , these look somewhat weird with `LibUtils_` prefix. Not sure if this is a good way.

Reply:
Maybe we can define new set of macros to use in all files in `lb/spinel` with totally different names  (similar to how we provide `otEXPECT` for use by platform).